xesar-connect/src/main/kotlin/com/open200/xesar/connect/messages/query/CodingStation.kt

```diff
@@ -0,0 +1,23 @@
+package com.open200.xesar.connect.messages.query
+
+import QueryElementResource
+import QueryListResource
+import com.open200.xesar.connect.utils.LocalDateTimeSerializer
+import com.open200.xesar.connect.utils.UUIDSerializer
+import kotlinx.serialization.Serializable
+import java.time.LocalDateTime
+import java.util.*
+
+@Serializable
+data class CodingStation(
+    @Serializable(with = UUIDSerializer::class) val id: UUID,
+    val name: String? = null,
+    val description: String? = null,
+    @Serializable(with = UUIDSerializer::class) val partitionId: UUID,
+    @Serializable(with = LocalDateTimeSerializer::class) val deleted: LocalDateTime? = null,
+    val online: Boolean? = null
```

Comment:
We need to check the fields on this and other resources if they are optional or required. I think they are similar to the Calendar where we have too many optional fields.